src/Okta.DNX.OAuth.ResourceServer/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Good point. The Readme is in the latest [commit](https://github.com/raphaellondner-okta/okta-oauth-dotnetcore-rs-simple/pull/1/commits/8a6190253df94fdfaa6b71c4f9142b1000eacc0a) but not sure it's enough